src/index.ts

```diff
@@ -1,5 +1,9 @@
-import { openBrowser } from "./helpers/browser";
+//import { openBrowser } from "./helpers/browser";
```

Comment:
Are you going to leave the comments here?

Reply:
I already removed.